lib/matplotlib/tests/test_axes.py

```diff
@@ -7387,3 +7387,12 @@ def test_clim():
         clim = (7, 8)
         norm = plot_method(clim=clim).norm
         assert (norm.vmin, norm.vmax) == clim
+
+
+@image_comparison(["extent_units"])
```

Comment:
Can you mark this to have the mpl2 style (so we do not test jet + interpolation)?

Reply:
and I suspect we can get away with _only_ the pdf or svg tests here (they are not affected by the freetype version issues, we do not not really want to test font rendering here, and any issues we are going to have with units are going to happen well _above_ the renderer level so generating the pdf/png/svg does not actually get us more meaningful test coverage.

Reply:
Will make the changes, thank you